src/Tests/AllDocsResourceTest.php

```diff
@@ -27,12 +28,14 @@ public function testGet() {
     $account = $this->drupalCreateUser($permissions);
     $this->drupalLogin($account);
 
-    $entities = array();
+    $entities = [];
     $entity_types = array('entity_test_rev');
+    $entity_manager = \Drupal::entityManager();
```

Comment:
Could we use `$this->entityManager` which is set in `ResourceTestBase::setUp()`?